### ISSUE TITLE
drop in assets -> show file doesn't work (macOS 3.4.1)

### ISSUE THREAD

**Issue by gorgatron1** [syd711/vpin-studio#430]
**Describe the bug**
drop in assets -> show file (the little folder icon at the far right) doesn't work (macOS 3.4.1)

**To Reproduce**

1. set up the drop in assets folder
2. put some tables in there
3. click on the drop in asset button
4. it shows the files and there is a folder icon on the far right
5. I expected this to open the file in the Finder (macOS file browser) but as far as I can tell it did nothing

<img width="2397" alt="image" src="https://github.com/user-attachments/assets/1ce23c56-16d9-49eb-ab7d-13d3a98ec82b">

**Expected behavior**

I expected it to open a Finder window like this -- not sure if that is what it is supposed to do, but it currently does nothing that I can see.

<img width="1032" alt="image" src="https://github.com/user-attachments/assets/b78ee0e4-6420-40d3-95bf-b91f7164e0a1">

Assuming this is correct, the command to do this on macOS is:

```
open -R ~/Downloads/pinball/Senna\ Prototype\ Edition\ \(Culik\ Pinball\ 2020\)\ VPX8.zip
```

from the man page:

```
NAME
     open – open files and directories

SYNOPSIS
     open [-e] [-t] [-f] [-F] [-W] [-R] [-n] [-g] [-j] [-h] [-u URL] [-s sdk] [-b bundle_identifier] [-a application]
          [--env VAR] [--stderr PATH] [--stdin PATH] [--stdout PATH] [--arch ARCH] [--args arg1 ...]

...
     -R  Reveals the file(s) in the Finder instead of opening them.
```

**Screenshots**

**Desktop (please complete the following information):**
 - OS: macOS 15.0

**Comment by syd711** [comment]
I assume I simply used the wrong "open" call.

**Comment by syd711** [comment]
Fixed with 3.6.0
